rules/group_rules.yaml

```diff
@@ -19,6 +19,10 @@
       # Must use quotes if the condition starts with @ symbol.
     - member_email: "@{DOMAIN}"
     - member_email: "@gmail.com"
+    # Service Accounts
+    - member_email: "gserviceaccount.com"
```

Comment:
Can we make this opt-in by default, i.e. by commenting this out?  Then if people want it, then can uncomment it.
```
# - member_email: "gserviceaccount.com"
```